### ISSUE TITLE
Feature request: the ability to cap a grade category at x points

### ISSUE THREAD

**Issue by dqnykamp** [Doenet/DoenetTools#1558]
We currently have the ability to calculate a grade category based on the top n scores.

In contrast, this request is an option where after calculating the total for the category (whether based on the top n or all scores), we calculate the total as the minimum of that score and the number x.

**Comment by Lyanthropos** [comment]
Would you expect this to be applied after the scale factor for the category? Otherwise you end up scaling the cap too.

**Comment by dqnykamp** [comment]
Right.  It should be applied after the scale factor so that the cap is not scaled.
